<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft Universal In-Game Timer | Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="19" column="5" colspan="2">
     <widget class="QRadioButton" name="light_theme_button">
      <property name="text">
       <string>Light</string>
      </property>
     </widget>
    </item>
    <item row="19" column="2" colspan="2" alignment="Qt::AlignRight">
     <widget class="QRadioButton" name="dark_theme_button">
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0" colspan="8">
     <widget class="QPushButton" name="continue_button">
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="theme_text">
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="browse_text">
      <property name="text">
       <string>Select your Minecraft directory, unless it's already filled in below:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="7">
     <widget class="QLineEdit" name="browse_field"/>
    </item>
    <item row="0" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="welcome_text">
      <property name="text">
       <string>Welcome to the Universal In-Game Timer for Minecraft!</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="browse_button">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2" colspan="5">
     <widget class="QSlider" name="opacity_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="opacity_min_text">
      <property name="text">
       <string>10%</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="opacity_text">
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
    </item>
    <item row="14" column="7">
     <widget class="QLabel" name="opacity_max_text">
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="author_text">
      <property name="text">
       <string>By NinjaSnail1080</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="8" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="browse_leave_empty_text">
      <property name="text">
       <string>(If left empty, it'll look for the default Minecraft directory)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
